pyiron_atomistics/atomistics/master/murnaghan.py

```diff
@@ -590,7 +591,7 @@ def pouriertarantola(V, E0, B0, BP, V0):
 
 
 # ToDo: not all abstract methods implemented
-class Murnaghan(AtomisticParallelMaster):
+class Murnaghan(AtomisticParallelMaster, HasStructure):
```

Comment:
I would really like that `AtomisticParallelMaster` has some `Atomistic` parent that already gives us the `HasStructure` inheritance. Doesn't have to happen in this PR though, and I haven't looked yet what the current tree is.

Reply:
I'll check the tree if it is necessary.